app/core/routers/document.py

```diff
@@ -58,7 +62,13 @@ async def get_preview(
     :return: 400 if there were invalid parameters, otherwise
     the requested file converted accordingly to pdf.
     """
-    if first_page >= 1 and (first_page <= last_page or last_page == 0):
+    validation_errors: Optional[Response] = check_for_document_metadata_errors(
+        first_page=first_page,
+        last_page=last_page,
```

Comment:
The default values for first_page and last_page are `1` and `0` and when the system checks this values it return `NUMBER_OF_PAGES_NOT_VALID`. Is this the intended behaviour?

Reply:
it returns None if there are no errors (aka 1 first page and 0 last page)

Reply:
@RakuJa explains the logic of the `check_for_document_metadata_errors` function and the default values are working as intended!